README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
I think pointing to one of the projects (or a new one) in https://github.com/atomify/atomify-examples would be easier to follow than this inline example.